src/js/components/MultiSelect/ColumnSelect.js

```diff
@@ -165,117 +167,128 @@ const ColumnSelect = ({
           width={layout === 'single-column' ? '100%' : '50%'}
           // pad={{ vertical: 'small' }}
         >
-          <OptionsBox role="menubar" tabIndex="-1">
-            {options.length > 0 ? (
-              <InfiniteScroll
-                items={getOptionsList()}
-                step={theme.select.step}
-                onMore={onMore}
-                replace={replace}
-                // show={activeIndex !== -1 ? activeIndex : undefined}
-              >
-                {(index, _itemIndex, optionRef) => {
-                  const optionDisabled = isDisabled(index);
-                  const optionSelected = isSelected(index);
-                  const optionActive = activeIndex === index;
-                  return (
-                    <>
-                      {index === 0 && showSelectAll && (
+          {!loading ? (
```

Comment:
Extract the codes inside the ternary operator to a function. Nested ternary operators makes it less readable.

Reply:
extracted out option list in separate function to make it more readable